src/betterproto/__init__.py

```diff
@@ -505,30 +545,54 @@ class Message(ABC):
     _serialized_on_wire: bool
     _unknown_fields: bytes
     _group_current: Dict[str, str]
+    __dataclass_fields__: Mapping[str, dataclasses.Field]  # Technically should be Final
```

Comment:
Don't think this comment is useful. Betterproto won't be dropping 3.7 support for a long time.

Reply:
I was thinking of PRing support for typing_extensions at some point to stop relying on class dunders, which supports `Final`, but I'll remove it for now.